### DISCUSSION TITLE
I think it is a Problem with vs code css and icons.

### DISCUSSION THREAD

**Discussion by abuhabban** [coder/code-server#4025] [General]
**Look below and then compare with these two `screenshots.`**

1. Screenshot captures from (**windows**).
![Screenshot_20210824_101635](https://user-images.githubusercontent.com/70461458/130593082-713dcc63-1ba7-445e-85c6-6ec63e2c72f9.jpg)

2. Screenshot captures from ( Android ).
![Screenshot_20210824_144144](https://user-images.githubusercontent.com/70461458/130593313-cc6e2996-8b73-4db1-b43f-5849ce069ecf.jpg)

You can see window screenshot looking good and bright but Android not . Why this is happening i don't know. I have try all devices but problem not solved.

**Comment by AyaanZaveri** [comment]
If this is happening with all your files (HTML, JS, etc), then it is probably a font issue. If you go to VS Code Settings (code-server) and change the font to something like Arial it should work.

**Reply by jsjoeio** [reply]
Thanks for chiming in @AyaanZaveri.

@abuhabban Let us know if this works!

**Reply by abuhabban** [reply]
`Arial fonts works Fine.`
But finall it works better when i erased all font.

Thanks all 
![Screenshot_20210825_061921](https://user-images.githubusercontent.com/70461458/130708603-aa2d8ded-0ce7-429a-a762-c93574782ce4.jpg)


**But i think left side all menus should be little more small just compare with window screenshot it look so fine if you are using Android.**
_It just a suggestion sir not advice_
Thank You.

**Reply by abuhabban** [reply]
Look at the screenshot icon is looking too bigger so it is creating a problem while typing. It cover's all screen with auto zoom.

![Screenshot_20210825_074142](https://user-images.githubusercontent.com/70461458/130715249-7f8cca52-666e-417d-b200-1195b7163607.jpg)

**Reply by lg-nineoneninethreenine** [reply]
It depends mainly on screen resolution. Since laptop screens are bigger, they produce much clearer images of code-server, but mobile is something that is supposed to fit inside your pocket.

So, mobile screens are _smaller_ than laptop screens. Even if the mobile and laptop screen ratio is same, the _sizes_ are **different**. 

If you take a phone and a laptop for code-server, the laptop can be used easily whereas, for the phone, you need external peripherals, or things like that. Point is, if you want to code on a phone, it's gonna be hard since code-server is mainly for laptops, desktops and iPads(they have real external peripherals)

**Reply by jsjoeio** [reply]
> Point is, if you want to code on a phone, it's gonna be hard since code-server is mainly for laptops, desktops and iPads(they have real external peripherals)

yeah @im-coder-lg is right :( 

I agree though, we could improve the Android/mobile experience. Who knows, maybe we write a custom VS Code extension which changes the size of things and makes it more user-friendly on mobile.

**Reply by lg-nineoneninethreenine** [reply]
Maybe, once, I was using Gitpod(needed quick env) and it was so hard just to open the terminal and I was suffering, literally, to just type `codedoc serve`. The keyboard continuously disappeared, so it was torturing. But, if code-server got updated to the latest update of VS Code, it has a feature to move the terminal to the editor area, so it might work. Only caveat: The people like me, who love the open Explorer, sorry, you can't do it in mobile.

**Reply by lg-nineoneninethreenine** [reply]
But, as usual, there are methods to make this work. 
1. Get an external display. If your phone is a Samsung that was released after/during 2019, it supports Samsung DeX so just plug the phone in and start code-server up.
2. Get external peripherals like keyboards and mouses(wireless than wired)
3. If you can't get code-server on your phone, get it on your laptop, and try using Cloudflare/ngrok(danger levels) and forward the 8080(your code-server port) port and access it from the phone.

Even if your phone doesn't have Samsung DeX or it's alikes, you can do some methods and use the phone from the external screen.

Phones running Android 10 have a hidden Desktop mode that can be used(?)

Or, you might have to wait till the next release of code-server which will surely include moving the terminal to the editor area.
